SingularityUI/app/components/machines/Racks.jsx

```diff
@@ -221,15 +202,43 @@ const Racks = React.createClass({
     <MachinesPage
       header = "Racks"
       states = {this.getStates()}
+      error = {this.props.error}
     />
     );
   }
 });
 
+function getErrorFromState(state) {
+  if (state.api.decommissionRack.error) {
```

Comment:
a `const { decommissionRack, removeRack, reactivateRack } = state.api;` might be nice here